### PR TITLE
Feature/amp

### PR DESCRIPTION
**Pull request by goldquest**
This adds a new subtheme to use AMP with Thunder

To test it, you have to activate the AMP module, install the `thunder_amp` theme and go to an article. Here you have to append `?amp` to the URL.
To validate the output you have different options. Please read about them here: https://www.ampproject.org/docs/guides/validate

Make sure these boxes are checked before submitting your pull request - thank you!

- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/8.x-1.x/docs/development.md#how-to-run-the-tests))
- [ ] ~Necessary update hooks are provided.~
- [ ] ~User roles have correct access for new introduced permission.~
- [x] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [x] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: config/optional/amp.theme.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+amptheme: thunder_amp
```
Comment (goldquest) [inline config/optional/amp.theme.yml]: This has no dependency at all. Wouldn't that be configured, even if I install something else?

File: thunder.profile [hunk 2]
```diff
@@ -232,18 +232,14 @@ function thunder_themes_installed($theme_list) {
...
-    $ampTheme = $ampThemeConfig->get('amptheme');
-    if (empty($ampTheme) || $ampTheme == 'ampsubtheme_example' || $ampTheme == 'amptheme') {
-      $ampThemeConfig->set('amptheme', 'thunder_amp')
-        ->save(TRUE);
-      \Drupal::configFactory()
-        ->getEditable('amp.settings')
-        ->set('amp_library_process_full_html', 1)
-        ->save(TRUE);
-    }
+    \Drupal::configFactory()
```
Comment (goldquest) [inline thunder.profile]: This can't be solved via optional config?

File: config/optional/amp.theme.yml (hunk 1 above)
Comment (chrfritsch) [inline config/optional/amp.theme.yml]: hm, that's true. if you install amp module without thunder_amp then you will also have this setting. which will be useless

File: thunder.profile (hunk 2 above)
Comment (chrfritsch) [inline thunder.profile]: no, sadly not

File: config/optional/amp.theme.yml (hunk 1 above)
Comment (goldquest) [inline config/optional/amp.theme.yml]: Do you think, we really need this as an optional config? We could let it be in the themes_installed hook. At least we have everything in the same spot then.

**Comment by chrfritsch** [comment]
- Images in the gallery have different heights and don't respect the focal point
- Instagrams are totally broken, could maybe fixed by using https://www.drupal.org/node/2807735

**Comment by chrfritsch** [comment]
- Under the gallery is a lot of whitespace

**Comment by goldquest** [comment]
I decided to use an existing Image Style. But AMP installs two image styles, which are not using focal point. I think it is ok to leave them there as the "default". What do you think?

**Comment by goldquest** [comment]
There is also the issue, that when you try to install AMP during the thunder installation process, the blocks aren't configured properly.
It would be expected, that most are diabled, but they only are, if the theme gets activated after installation is finished.

**Comment by goldquest** [comment]
We maybe need a warning / requirement check, when installing the AMP theme / module later (Not as optional on installation)

**Pull request merged**